src/tests/ftest/network/zero_config.py

```diff
@@ -143,6 +143,9 @@ def verify_client_run(self, exp_iface, env):
         # Add FI_LOG_LEVEL to get more info on device issues
         racer_env = daos_racer.get_environment(self.server_managers[0], logf)
         racer_env["FI_LOG_LEVEL"] = "info"
+        # bump/double log size limit to avoid log switch and be able to
+        # find initialization msg in single/first log file
+        racer_env["D_LOG_SIZE"] = "2G"
```

Comment:
Is this the correct fix here or should the test be aware of the `.old` log possibility and check that log (first, since it's looking for the beginning of the log, isn't it?) if it exists?

Reply:
This is the simplest fix! And anyway, raising the size may become necessary if the log verbosity raises and will allow to exceed 2x1GB ...

Reply:
> This is the simplest fix! 

Indeed, it is the simplest.  But simplest is not always the correct fix.

> And anyway, raising the size may become necessary if the log verbosity raises and will allow to exceed 2x1GB ...

At which point I'd say that the test failing is just a symptom of the larger problem of having excessively large logs with low quality.  At that point if large log files is causing that test to fail, the fix is to address the log size, not just keep allowing them to be bigger and bigger, just to pass tests.

Reply:
> At that point if large log files is causing that test to fail, the fix is to address the log size

You mean the reason why the log is growing too much ? Then this is not the object of this ticket/PR right ?

Reply:
Yes.  This PR isn't to address reasons why the log grows, but just to limit the size of a log and the ripple effects caused by excessive logs.  But what I am saying is that if the log grows beyond the default limit of 2G (1G + 1G .old) and that affects a test (which reads the log, say) or debugging efforts, then something needs to be done to address the size (i.e. quality) of the log.

But getting back to reason for this conversation, the problem with the "fix" https://github.com/daos-stack/daos/blob/d40c16eda08fb4312cf6a65677147033b9f8093e/src/tests/ftest/network/zero_config.py#L148
is that test is reading the log and wants to read what's at the beginning of it.  But due to this PR, the beginning of the log could be in the `.old` log file.  So the fix for that is to make the test know that it needs to read from the `.old` log file, if it exists, rather than just allowing the test to create bigger logs so that a `.old` is not generated.